docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
+Percona Alerting is powered by Grafana infrastructure. PMM leverages Grafana's advanced alerting capabilities and adds an extra layer of alert templates that simplifies complex alert rules.
+Depending on the datasources that you want to query, and the complexity of your required evaluation criteria, PMM enables you to create the following types of alerts:
 
-- **Percona templated alerts**: alerts based on a set of default templates with common events and expressions for alerting. 
-If you need custom expressions on which to base your alert rules, you can also create your own templates. 
+- **Percona templated alerts**: alerts based on a set of default templates with common events and expressions for alerting.
+If you need custom expressions on which to base your alert rules, you can also create your own templates.
 - **Grafana managed alerts**: alerts that handle complex conditions and can span multiple different data sources like SQL, Prometheus, InfluxDB, etc. These alerts are stored and executed by Grafana.
+<!--- we dont use support these for now so commenting them out
+
 - **Mimir or Loki alerts**: alerts that consist of one single query, written in PromQL or LogQL. The alert rules are stored and executed on the Mimir or Loki ruler and are completely decoupled from the PMM and Grafana runtime.
-- **Mimir or Loki recording rules**: precompute the result of expensive queries and execute alerts faster. 
-With Mimir and Loki alert rules, you can run alert expressions closer to your data and at massive scale, managed by Grafana. 
+- **Mimir or Loki recording rules**: precompute the result of expensive queries and execute alerts faster.
+With Mimir and Loki alert rules, you can run alert expressions closer to your data and at massive scale, managed by Grafana.
 
-## Alerting components
-Alerts are split into four key components: alert rules, contact points, notification policies, and silences. 
+-->
 
-### Alert rules
-Describe the circumstances under which you want to be alerted. The evaluation criteria that you define determine whether an alert will fire. 
+## Alerting components
 
-An alert rule consists of one or more queries and expressions, a condition, the frequency of evaluation, and the duration over which the condition is met.
+Alerts are split into four key components: alert rules, contact points, notification policies, and silences.
 
-For example, you might configure an alert to identify and notify you when MongoDB is down.
+### Alert rules
 
-Provide a simplified framework for configuring complex alert rules. 
+Alert rules describe the circumstances under which you want to be alerted. The evaluation criteria that you define determine whether an alert will fire. An alert rule consists of one or more queries and expressions, a condition, the frequency of evaluation, and the duration over which the condition is met. For example, you might configure an alert to identify and notify you when MongoDB is down.
```

Comment:
"you might configure an alert to **fire** and notify you when MongoDB is down (we should be consistent with the terminology used on the Percona Alerting pages)"